docs/pages/setup/reference/metrics.mdx

```diff
@@ -108,6 +108,7 @@ Now you can see the monitoring information by visiting several endpoints:
 | `reversetunnel_connected_proxies` | gauge | Teleport | Number of known proxies being sought. |
 | `rx` | counter | Teleport | Number of bytes received. |
 | `server_interactive_sessions_total` | gauge | Teleport | Number of active sessions. |
+| `teleport_connected_resources` | gauge | Teleport Auth | Tracks the number and type of resources connected via keepalives. |
```

Comment:
@rcanderson23 What kind of resources does this refer to specifically? Reverse tunnel agents or something else? Just trying to see if we can come up with a more clear metric name.

Reply:
@r0mant Resources types that create a keepalive with the auth server (whether through a reverse tunnel or direct). At the moment that would `node`, `db`, `app`, and `windows_desktop_service`. Example of the metric with my test cluster:
```
# HELP teleport_connected_resources Tracks the number and type of resources connected via keepalives
# TYPE teleport_connected_resources gauge
teleport_connected_resources{type="app"} 2
teleport_connected_resources{type="db"} 2
teleport_connected_resources{type="node"} 2
teleport_connected_resources{type="windows_desktop_service"} 1
```
This cluster has a teleport agent connected configured for two apps, another with two dbs configured, one node connected via reverse tunnel, one node connected directly, and one windows service. There are also two kube agents connected but kubernetes service currently doesn't use keepalive but would show up if/when https://github.com/gravitational/teleport/pull/9603#:~:text=using%20keepalive%20heartbeats%3B-,%239584,-will%20allow%20kube is merged.

Reply:
@rcanderson23 Sorry about delay on this one. So if this basically keeps track of the connected agents, would `connected_agents` be a better metric name?

I also think `teleport_` prefix is redundant, none of the other Teleport-specific metrics seem have it.

Reply:
For me, `agents` seems to be used synonymously with `servers` which is the reason I chose resources here. If you feel strongly about that, I can replace `resources` with `agents`. 

The reason for prefixing with Teleport is ease of discoverability in Prometheus and Grafana for users and most applications prefix their application metrics like this. New metrics I have added have had this prefix as well.

Reply:
The common exception to this prefixing would be common metrics from libraries such as the grpc metrics from https://github.com/grpc-ecosystem/go-grpc-prometheus